src/Microsoft.DocAsCode.MarkdownLite/JsonTokenTreeRenderer.cs

```diff
@@ -273,6 +278,15 @@ private string GetSize()
             return "1";
         }
 
+        private string escape(string content)
+        {
+            return content.Replace("\n", "\\n")
+                .Replace("\"", "&quot;")
+                .Replace("<", "&lt;")
+                .Replace(">", "&gt;")
+                .Replace("'", "&#39;");
```

Comment:
But it can't replace the "\n" for that "\n" will break the structure of json

---
In reply to: [75988652](https://github.com/dotnet/docfx/pull/631#discussion_r75988652) [](ancestors = 75988652)